vyper/venom/analysis/mem_ssa.py

```diff
@@ -86,7 +86,7 @@ class MemoryPhi(MemoryAccess):
     def __init__(self, id: int, block: IRBasicBlock):
         super().__init__(id)
         self.block = block
-        self.operands: list[tuple[MemoryDef, IRBasicBlock]] = []
+        self.operands: list[tuple[MemoryAccess, IRBasicBlock]] = []
```

Comment:
This can be `MemoryDef ` or `MemoryPhi`, should we make a type alias for the two?

Reply:
i'm not a fan of type aliases in general, but i'll change the signature

Reply:
dd791da1a8e7d5cfdb90dd0dcc1b370abcfd0a2b